### PR TITLE
navmazing convertions

### PR DESCRIPTION
**Pull request by gshefer**
conversion to navmazing
{{pytest: cfme/tests/containers/test_containers_projects_summary.py -v --use-provider container }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a5c8b29bf3c14d10c53441e05b7a6cd326875eb4:

`cfme/tests/containers/test_containers_projects_summary.py`:
- :red_circle: Line 6:1: **F401** *'tb' imported but unused*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 491ce5f2afb9ed250e670a94f45aa76e7f5d8244:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c10e49b764f6d3238516d18fcb276c2f83f37bd8:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 488c0e08f017efecf9a32b8969320890d00d2760:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/containers/pod.py [hunk 1]
```diff
@@ -52,9 +42,28 @@ def get_detail(self, *ident):
...
+@navigator.register(Pod, 'All')
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
+
+    def step(self):
+        nav._nav_to_fn('Compute', 'Containers', 'Pods')(None)
+
+    def resetter(self):
+        # Reset view and selection
+        tb.select("List View")
```
Comment (RonnyPfannschmidt) [inline cfme/containers/pod.py]: future note, we should mark this for the widgettastig convert later on

File: cfme/containers/pod.py [hunk 2]
```diff
@@ -1,41 +1,31 @@
...
-            {'Name': ctx['pod'].name, 'Provider': ctx['pod'].provider.name}),
-    }
-)
 
+class Pod(Taggable, SummaryMixin, Navigatable):
 
-class Pod(Taggable, SummaryMixin):
-
-    def __init__(self, name, provider):
+    def __init__(self, name, provider, applience=None):
```
Comment (psav) [inline cfme/containers/pod.py]: appliance is misspelt here

File: cfme/containers/replicator.py [hunk 3]
```diff
@@ -2,39 +2,30 @@
...
-            {'Name': ctx['replicator'].name, 'Provider': ctx['replicator'].provider.name}),
-    }
-)
 
+class Replicator(Taggable, SummaryMixin, Navigatable):
 
-class Replicator(Taggable, SummaryMixin):
-
-    def __init__(self, name, provider):
+    def __init__(self, name, provider, applience=None):
```
Comment (psav) [inline cfme/containers/replicator.py]: spelling...

File: cfme/containers/pod.py (hunk 1 above)
Comment (gshefer) [inline cfme/containers/pod.py]: Cool - this line appears in many places so it'll be easy to re-factor this later on

**Comment by dajoRH** [comment]
Lint report for commit 8f25b35b320370c09fcc509ca8986f0e50d8d89f:

`cfme/tests/containers/test_views_objects.py`:
- :bangbang: Line 46:5: **E265** *block comment should start with '# '*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bc2cde40692c0d1b7422550a856f72b83efe70a0:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit add25e97299cb7f24fb49a5c01c22fcd4694adf3:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ee4945742195ceda239f0110450f27792aaebbca:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6742a13bbebc6c52a2863d37e56790e6b4ee9ad2:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**